tox.ini

```diff
@@ -1,25 +1,35 @@
 [tox]
-envlist = py33, py34, py35, flake8
+envlist = py27, py33, py34, py35, flake8
 
 [travis]
 python =
     3.5: py35
     3.4: py34
     3.3: py33
+    2.7: py27
 
 [testenv:flake8]
 basepython=python
 deps=flake8
 commands=flake8 partridge
 
 [testenv]
+basepython =
```

Comment:
I don't think this is necessary, is it?

Reply:
You are correct. This has been removed for all env.